src/pcm/SoxrResampler.cxx

```diff
@@ -80,20 +88,110 @@ soxr_parse_quality(const char *quality) noexcept
 	return SOXR_INVALID_RECIPE;
 }
 
+static unsigned 
+soxr_parse_precision(unsigned value) {
+	switch(value) {
+		case 16:
```

Comment:
Bad coding style, see https://www.musicpd.org/doc/html/developer.html#code-style
This problem affects most of your code!

Reply:
Was not sure to if I should follow the code-style or the style of the already existing code surrounding the code change. Ok clear always use the code-style.